department-of-veterans-affairs/caseflow
# Khalin/appeals 29425

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/APPEALS-29425)

# Description
- The Mail team user is able to access a section that displays information selected in the "Associate with Prior Mail" section in step 1 of the correspondence intake form 
- The mail user is able to click a link titled "Edit Section"

## Acceptance Criteria

## Testing Plan

1. First run the following command to ensure the correspondence.rb file has seeded the relevant data for testing - `bundle exec rake db:seed:correspondence`
2. In rails console pull the uuid for the .last created correspondence  
3. Open browser and got to http://localhost:3000/queue/correspondence/:correspondence_uuid/intake use the UUID found in the rails console for previous step in the link
4. Click the "Yes" radio button under the Associate with prior Mail section
5. Select 2-3 check boxes for the prior mail to link to this correspondence 
6. "Continue" to go to step 2
7. "Continue" to go to step 3
8. The Table in AC1 should be on the page
9. Click the edit section link to go back to step 1
# Frontend
## User Facing Changes

<img width="1250" alt="Screenshot 2023-11-29 at 4 33 00 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/134650238/ccc4afea-9e07-40cc-be09-b90bc78d70e5">


## Storybook Story
*For Frontend (Presentation) Components*
